core/network/address.go

```diff
@@ -477,10 +477,10 @@ func (sas SpaceAddresses) ToProviderAddresses(lookup SpaceLookup) (ProviderAddre
 		return nil, nil
 	}
 
-	var infoFor map[string]SpaceInfo
+	var infoFor SpaceInfos
```

Comment:
I predicated this name on the map index lookup. I think we can now just call it "spaces" or some such.

Reply:
done